projects/gloo/api/v1/settings.proto

```diff
@@ -287,7 +287,7 @@ message Settings {
 
         message FdsOptions {
             // Enable function discovery service on GraphQL gRPC and OpenApi upstreams. Defaults to true.
```

Comment:
we should also add language about deprecation to the field comments; i don't think the `[deprecated = true]` tag has any effect on our generated docs.

Reply:
Good catch, updated text